src/test/java/TasksServletTest.java

```diff
@@ -116,8 +120,81 @@
           TASK_COMPLETED_YESTERDAY,
           TASK_COMPLETED_TODAY);
 
+  private static final TasksResponse NO_TASKLISTS_RESPONSE =
+      TasksResponse.builder()
+          .taskListTitles(NO_TASKLISTS_TITLES)
+          .tasksToComplete(0)
+          .tasksDueToday(0)
+          .tasksCompletedToday(0)
+          .tasksOverdue(0)
+          .build();
+  private static final TasksResponse NO_TASKS_RESPONSE =
+      TasksResponse.builder()
+          .taskListTitles(ONE_TASKLIST_TITLES)
+          .tasksToComplete(0)
+          .tasksDueToday(0)
+          .tasksCompletedToday(0)
+          .tasksOverdue(0)
+          .build();
+  private static final TasksResponse TASKS_DUE_YESTERDAY_RESPONSE =
+      TasksResponse.builder()
+          .taskListTitles(ONE_TASKLIST_TITLES)
+          .tasksToComplete(1)
+          .tasksDueToday(0)
+          .tasksCompletedToday(0)
+          .tasksOverdue(1)
+          .build();
+  private static final TasksResponse TASKS_DUE_TODAY_RESPONSE =
+      TasksResponse.builder()
+          .taskListTitles(ONE_TASKLIST_TITLES)
+          .tasksToComplete(1)
+          .tasksDueToday(1)
+          .tasksCompletedToday(0)
+          .tasksOverdue(0)
+          .build();
+  private static final TasksResponse TASKS_DUE_TOMORROW_RESPONSE =
+      TasksResponse.builder()
+          .taskListTitles(ONE_TASKLIST_TITLES)
+          .tasksToComplete(1)
+          .tasksDueToday(0)
+          .tasksCompletedToday(0)
+          .tasksOverdue(0)
+          .build();
+  private static final TasksResponse TASKS_COMPLETED_YESTERDAY_RESPONSE =
+      TasksResponse.builder()
+          .taskListTitles(ONE_TASKLIST_TITLES)
+          .tasksToComplete(0)
+          .tasksDueToday(0)
+          .tasksCompletedToday(0)
+          .tasksOverdue(0)
+          .build();
+  private static final TasksResponse TASKS_COMPLETED_TODAY_RESPONSE =
+      TasksResponse.builder()
+          .taskListTitles(ONE_TASKLIST_TITLES)
+          .tasksToComplete(0)
+          .tasksDueToday(0)
+          .tasksCompletedToday(1)
+          .tasksOverdue(0)
+          .build();
+  private static final TasksResponse ALL_TASKS_RESPONSE =
+      TasksResponse.builder()
+          .taskListTitles(ONE_TASKLIST_TITLES)
+          .tasksToComplete(3)
+          .tasksDueToday(1)
+          .tasksCompletedToday(1)
+          .tasksOverdue(1)
+          .build();
+  private static final TasksResponse MULTIPLE_TASKLISTS_RESPONSE =
+      TasksResponse.builder()
+          .taskListTitles(TWO_TASKLISTS_TITLES)
+          .tasksToComplete(6)
+          .tasksDueToday(2)
+          .tasksCompletedToday(2)
+          .tasksOverdue(2)
+          .build();
```

Comment:
https://github.com/googleinterns/step7-2020/pull/88#discussion_r459100105
> It's not immediately obvious that these responses are correct, just by looking at this test method. I see that you have incomplete and overdue tasks, but how do I know that `TASKS_DUE_YESTERDAY` only returns one overdue task?
> 
> If you follow my suggestion above (to allow comparing responses), then you can define static constants for the expected responses to each set of input tasks. This would put the definition of the responses closer to where the inputs are defined.
> 
> (I am not sure what you would do with more complex tests that use multiple task lists - probably just define responses for those combinations.)

Reply:
`RESOLVED`